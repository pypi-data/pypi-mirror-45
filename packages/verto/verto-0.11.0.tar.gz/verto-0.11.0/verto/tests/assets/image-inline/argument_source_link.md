An inline image: {image-inline file-path="http://placehold.it/350x150" alt="alt text" source="http://www.example.com"}.
