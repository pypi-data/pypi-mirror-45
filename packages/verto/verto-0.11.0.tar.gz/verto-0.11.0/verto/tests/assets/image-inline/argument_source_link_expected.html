<p>An inline image: <div>
<img alt="alt text" src="http://placehold.it/350x150" />
<p><a href="http://www.example.com">Source</a></p>
</div>.</p>
